kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
 
 [^90]: पुष्पाणां प्रवरा जाती धूपानां कुन्दको वरः। गन्धानां कुङ्कमं श्रेष्ठं लेपानां चन्दनं वरम् ॥ दीपदाने घृतं श्रेष्ठं मोदकाश्च निवेदने। भविष्यपुराण (ब्राह्मपर्व 198. 19 q. by कल्पतरु on व्रत p. 183 ).
 
-[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्श्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.
+[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव पिडाङ्श्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.
```

Comment:
First correction is doubtful - can you send a screenshot on Telegram plz?

Reply:
at present I have put पिडाङ्श्चैव{??} like this. I will correct it once we get the correct word.

Reply:
I looked up garuDapurANa. Please put विडङ्गश्चैव

Reply:
corrected.
[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्गश्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.

Reply:
corrected. [^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्गश्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.